### PR TITLE
WRP-12532: Increase PopupTabLayout automated tests.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to add ui-test for PopupTabLayout.
- Need to add new ui-test view for PopupTabLayout.
- Need to add ss-test for PopupTabLayout.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Add ui-test for PopupTabLayout.
- Add new ui-test view for PopupTabLayout.
- Add ss-test for PopupTabLayout.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
WRP-12532
### Comments
The reason why the panel of ui-test is open from the beginning is that we are testing the panel to be opened in the existing test. e.g)QWTC-656

### CODE REVIEW

File: tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	const handleKeyDown = () => (ev) => {
+		const {keyCode} = ev;
+
+		if (!rtl() && isRight(keyCode)) {
+			setIndex(1);
+		}
+	};
+
+	return (
+		<div>
```
Comment (sjro) [inline tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js:41]: I think <div></div> seems unnecessary

File: tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import {useState} from 'react';
+import Group from '@enact/ui/Group';
+import ilib from '@enact/i18n';
+import {is} from '@enact/core/keymap';
+import spotlight from '@enact/spotlight';
+
```
Comment (sjro) [inline tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js:6]: Please change the import order.
import {is} from '@enact/core/keymap';
import ilib from '@enact/i18n';
import spotlight from '@enact/spotlight';
import Group from '@enact/ui/Group';
import {useState} from 'react';

File: tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import {Header} from '../../../../../Panels';
+import Item from '../../../../../Item';
```
Comment (sjro) [inline tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js:8]: Please change the import order
import Item from '../../../../../Item';
import {Header} from '../../../../../Panels';

File: tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js (hunk 1 above)
Comment (dongsuwon) [reply tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js]: Suggested Changes.

File: tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js (hunk 2 above)
Comment (dongsuwon) [reply tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js:6]: Suggested Changes.

File: tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js (hunk 3 above)
Comment (dongsuwon) [reply tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js:7]: Suggested Changes.

**Review by sjro — APPROVED** [review]

**Pull request merged**